### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by JosuaKrause**
- show name in pretty
- box drawing in pretty

### CODE REVIEW

File: accern_xyme/accern_xyme.py [hunk 1]
```diff
@@ -1303,27 +1320,55 @@ def draw() -> List[str]:
             for node in order:
                 node_line = \
                     f"{node.get_short_status(allow_unicode)} " \
-                    f"{node.get_type()}[{node.get_id()}] " \
+                    f"{node.get_type()}[{node.get_name()}] " \
```
Comment (JosuaKrause) [inline accern_xyme/accern_xyme.py:1323]: swap type and name

File: accern_xyme/accern_xyme.py [hunk 2]
```diff
@@ -1303,27 +1320,55 @@ def draw() -> List[str]:
             for node in order:
                 node_line = \
```
Comment (JosuaKrause) [inline accern_xyme/accern_xyme.py:1321]: indent the line until the first actual connection

**Comment by JosuaKrause** [comment]
```
> print(pipe.pretty(nodes_only=True))
┌───┐
│ = ├────────────┐
└─┬─┘            │
  ├ jsons        ├ jsons
  │              ├ jsons_in (0)
  │            ┌─┴─┐
  │            │ = ├────────┬──────────┬─────────────┐
  │            └─┬─┘        │          │             │
  │              ├ data     ├ data     ├ data        ├ data
  │              │          │          │             ├ data (0)
  │              │          │          │           ┌─┴─┐
  │              │          │          │           │ = │
  │              │          │          │           └─┬─┘
  │              │          │          │             ├ embedding
  │              │          │          ├ data_in (0) ├ embedding (0)
  │              │          │        ┌─┴─┐           │
  │              │          │        │ = ├───────────┘
  │              │          │        └─┬─┘
  │              │          │          ├ data_out
  ├ jsons_in (0) │          │          ├ data (0)
  │              │          │        ┌─┴─┐
  └──────────────────────────────────┤ = ├────────────┐
                 │          │        └─┬─┘            │
                 │          │          ├ jsons        ├ jsons
                 │          ├ data (0) │              ├ jsons_in (0)
                 │        ┌─┴─┐        │              │
                 │        │ = ├───────────────────────┘
                 │        └───┘        │
                 │                     │
                 ├ data (0)            ├ jsons_in (0)
               ┌─┴─┐                   │
               │ = ├───────────────────┘
               └───┘
```
```
> print(pipe.pretty(nodes_only=False))
┌────────────────┐
│= in(jsons_in) 0├┐
└──┬─────────────┘│
   ├ jsons        ├ jsons
   │              ├ jsons_in (0)
   │           ┌──┴────────────────────────┐
   │           │= cd(custom_json_to_data) 0├──────────┐
   │           └──┬──────────┬──────────┬──┘          │
   │              ├ data     ├ data     ├ data        ├ data
   │              │          │          │             ├ data (0)
   │              │          │          │          ┌──┴──────────────────────────────┐
   │              │          │          │          │= embed(static_embedding_model) 0│
   │              │          │          │          └──┬──────────────────────────────┘
   │              │          │          │             ├ embedding
   │              │          │          ├ data_in (0) ├ embedding (0)
   │              │          │       ┌──┴─────────────┴────────┐
   │              │          │       │= sent(titan_sentiment) 0│
   │              │          │       └──┬──────────────────────┘
   │              │          │          ├ data_out
   ├ jsons_in (0) │          │          ├ data (0)
   │              │          │       ┌──┴────────────────────────────┐
   └─────────────────────────────────┤= cjjd(custom_json_join_data) 0│
                  │          │       └──┬──────────────┬─────────────┘
                  │          │          ├ jsons        ├ jsons
                  │          ├ data (0) │              ├ jsons_in (0)
                  │       ┌──┴─────────────────────────┴────┐
                  │       │= finfin(custom_json_join_data) 0│
                  │       └─────────────────────────────────┘
                  │                     │
                  ├ data (0)            ├ jsons_in (0)
               ┌──┴─────────────────────┴─────┐
               │= fin(custom_json_join_data) 0│
               └──────────────────────────────┘
```

**Review by jckhang — APPROVED** [review]
lgtm

**Pull request merged**